README.md

```diff
@@ -181,7 +181,33 @@ Upload benchmarks are from an Ubuntu 20.04 DNAnexus worker mem2_ssd1_v2_x32 (AWS
 |	37.8  |	87 | 10GiB |
 |	254  |	495 | 100GiB |
 
-## TBA
+
+# AllowOverwrite Mode
+If `-limitedWrite` and `-allowOverwrite` flags are both set when launching dxfuse, overwriting existing files (both created before or during the current session) is enabled, but **only in truncate mode** (with `O_TRUNC` flag at system call level)
+
+Sample use cases:
+- In terminal, write to an existing file by redirecting using '>' or 'tee'
+- In terminal, overwrite an existing file by `cp` or `dd`
+- write and save a file in RStudio editor
+- overwrite an existing file using R commands from an R script or in R console, for example:
+  - base::writeLines()
+  - data.table::fwrite()
+  - readr::write_rds()
+  - saveRDS()
+  - write.table()
+
+Other operations enabled by `-limitedWrite` mode are also supported. However, any other operations that modifies an existing file content without opening in truncate mode is **not supported**, for example:
+- write to an existing file by redirecting using '>>' in terminal
+- open existing file with `O_APPEND` flag
```

Comment:
```suggestion
- Opening an existing file with `O_APPEND` flag
```

Reply:
@YuxinShi0423 what if O_TRUNC and O_APPEND are both specified?  Is that possible?